oda_api/cli.py

```diff
@@ -72,6 +73,16 @@ def get(obj, instrument, product, argument, _discover_token):
                 for du in p.data_unit:
                     logger.info(">> %s", du.data)
 
+            for k, v in plot_tools.__dict__.items():
+                if not k.startswith('Oda'): continue
```

Comment:
why not just
```suggestion
                if k.startswith('Oda'):
```

Reply:
well it gets kind of deep. Though I had more stuff before, it's shorter now. Ok I changed